ephios/static/ephios/js/group_form.js

```diff
@@ -1,11 +1,31 @@
 $(document).ready(function () {
-    $("#id_can_add_event").click(function (event) {
+    $("#id_is_planning_group").click(function (event) {
         if ($(this).is(":checked"))
             $(".publish-select").slideDown();
         else
             $(".publish-select").slideUp();
     });
-    if (!$("#id_can_add_event").is(":checked")) {
+    if (!$("#id_is_planning_group").is(":checked")) {
         $(".publish-select").slideUp();
     }
+    $("#id_is_management_group").click(function (event) {
+        if ($(this).is(":checked")) {
+            $("#id_is_hr_group").prop("disabled", true);
+            $("#id_is_hr_group").prop("checked", true);
+            $("#id_is_planning_group").prop("disabled", true);
+            $("#id_is_planning_group").prop("checked", true);
+            $(".publish-select").slideDown();
+        }
+        else {
+            $("#id_is_hr_group").prop("disabled", false);
+            $("#id_is_hr_group").prop("checked", false);
+            $("#id_is_planning_group").prop("disabled", false);
+            $("#id_is_planning_group").prop("checked", false);
+            $(".publish-select").slideUp();
```

Comment:
can we have one call for the slideUp/Down at the end of the function based on the state of `is_planning_group`?

Reply:
and one single handler for all checkboxes? I think it would be nicer:

```javascript
function handleGroupForm(event) {
    // do the sliding and checking based on :checked
    // do the sliding instantly if event is null
}

$(() => {
    handleGroupForm();
    $(".groupformcheckbox").click(handleGroupForm);
});
```

Reply:
i don't understand what i should do here

Reply:
No problem, have a look at #360